packages/docs/operations-manual/key-management/key-rotation.md

```diff
@@ -0,0 +1,40 @@
+## Key Rotation
+
+If an authorized signer key is lost or compromised, the Locked Gold Account can authorize a new signer to replace the old one. This prevents losing an authorized signer key from becoming a catastrophic event. It is recommended to regularly rotate keys to limit the impact of keys being silently compromised.
+
+### Validator Signer Rotation
+
+Because the Validator signer key is constantly in use to sign consensus messages, special care must be taken when authorizing a new Validator signer. The following steps detail the recommended procedure for rotating the validator signer of an active and elected validator:
+
+1. Create a new Validator instance as detailed in the [Deploy a Validator](../getting-started/running-a-validator-in-baklava.md#deploy-a-validator) section of the getting started documentation. When using a proxy, additionally create a new proxy and peer it with the new validator instance, as described in the same document. Wait for the new instances to sync before proceeding.
+
+  {% hint style="warning" %}
+  Before proceeding to step 2 ensure there is sufficient time until the end of the epoch to complete key rotation.
+  {% endhint %}
+
+2. Authorize the new Validator signer key with the Locked Gold Account to overwrite the old Validator signer key.
+
+  ```bash
+  # With $SIGNER_TO_AUTHORIZE as the new validator signer:
+
+  # On the new validator node which contains the Locked Gold Account key
+  docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE --nousb account proof-of-possession $SIGNER_TO_AUTHORIZE $LOCKED_GOLD_ACCOUNT
+  docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE --nousb account proof-of-possession $SIGNER_TO_AUTHORIZE $LOCKED_GOLD_ACCOUNT --bls
+
+  # On a node containing your Locked Gold Account key.
+  celocli account:authorize --from $LOCKED_GOLD_ACCOUNT --role validator --signer $SIGNER_TO_AUTHORIZE --signature $SIGNER_PROOF_OF_POSSESSION --blsKey $BLS_PUBLIC_KEY --blsPop $BLS_PROOF_OF_POSSESSION
+  ```
+
+3. **Leave all validator and proxy nodes running** until the next epoch change. At the start the next epoch, the new Validator signer should take over participation in consensus.
+
+4. Verify that key rotation was successful. Here are some ways to check:
+  <!-- TODO: The following URL assumes that the user is running against the baklava network. This will need to be updated -->
+  * Open `baklava-blockscout.celo-testnet.org/address/<SIGNER_TO_AUTHORIZE>/validations` to confirm that blocks are being proposed.
```

Comment:
When mainnet launched: Thoughts on how to communicate this info for Baklava testnet w/out too much redundancy?